app/about/data-extraction/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+import { useContext, useEffect } from "react";
+import {
+  AboutPageSidebarOption,
+  aboutPageSidebarContext,
+} from "../about-page-context";
+
+export default function DataExtractionPage() {
+  const { setCurrentSidebarOption } = useContext(aboutPageSidebarContext);
+
+  useEffect(() => {
```

Comment:
For this and all the following pages, why do we have a useEffect? what is this a side effect of?

IF this is to set the context when a user navigates to this page, then IMO that should happen when the button is clicked not when this page loads.